gui-agent/vmside.c

```diff
@@ -1505,54 +1504,74 @@ static void mkghandles(Ghandles * g)
     g->screen = DefaultScreen(g->display);	/* get CRT id number */
     g->root_win = RootWindow(g->display, g->screen);	/* get default attributes */
     g->context = XCreateGC(g->display, g->root_win, 0, NULL);
-    g->wmDeleteMessage = XInternAtom(g->display, "WM_DELETE_WINDOW", False);
-    g->wmProtocols = XInternAtom(g->display, "WM_PROTOCOLS", False);
-    g->wm_hints = XInternAtom(g->display, "WM_HINTS", False);
-    g->wm_class = XInternAtom(g->display, "WM_CLASS", False);
-    g->utf8_string_atom = XInternAtom(g->display, "UTF8_STRING", False);
     g->stub_win = XCreateSimpleWindow(g->display, g->root_win,
             0, 0, 1, 1,
             0, BlackPixel(g->display,
                 g->screen),
             WhitePixel(g->display,
                 g->screen));
+    if ((unsigned)snprintf(tray_sel_atom_name, sizeof(tray_sel_atom_name),
+            "_NET_SYSTEM_TRAY_S%u", DefaultScreen(g->display)) >=
+        sizeof tray_sel_atom_name)
+        abort();
+#define SUPPORTED_ATOMS 6
+    const struct {
+        Atom *const dest;
+        const char *const name;
+    } atoms_to_intern[] = {
+        { &g->wmDeleteMessage,  "WM_DELETE_MESSAGE" },
```

Comment:
`WM_DELETE_WINDOW`
Can you @DemiMarie recheck other entries and compare with the old code (not with the comments, which apparently are wrong in some places)?

Reply:
I checked and that was the only one I could see.  I fixed both the variable name and comment to prevent similar problems in the future.  I also found a case where we were wastefully interning three atoms twice; that will be a separate patch since it is not urgent.